proxy/http/remap/NextHopSelectionStrategy.cc

```diff
@@ -63,6 +63,8 @@ bool
 NextHopSelectionStrategy::Init(ts::Yaml::Map &n)
 {
   NH_Debug(NH_DEBUG_TAG, "calling Init()");
+  std::string self_host;
```

Comment:
@ywkaras Consider adding this new field to the unit testing, see proxy/remap/http/unit-tests/test_NextHopConsistentHash.cc